### PR TITLE
Another mootools removal (languages overrides)

### PR DESCRIPTION
**Pull request by dgrammatiko**
#### Remove mootools from admin languages overrides view

Also the overrider.js got a compressed version
#### Testing

Apply patch and go to `administrator/index.php?option=com_languages&view=overrides`
Create a new override
on the right side insert `COM_CONTENT_` select constant and press search
at the end of the list there is a more link try that
now select something (should be copied to the left part)
Repeat for the values part e.g. `Article`
Also try without any value

### CODE REVIEW

File: administrator/components/com_languages/views/override/tmpl/edit.php [hunk 1]
```diff
@@ -17,9 +17,15 @@
...
+
+JHtml::_('behavior.core');
+JHtml::_('jquery.framework');
+JHtml::_('script', 'overrider/overrider.min.js', false, true, false, true);
+
 JFactory::getDocument()->addScriptDeclaration('
-	jQuery(document).ready(function() {
-		document.getElementById("jform_searchstring").addEvent("focus", function() {
+	jQuery(document).ready(function($) {
+		$("#jform_searchstring").on("focus", function() {
```
Comment (wilsonge) [inline administrator/components/com_languages/views/override/tmpl/edit.php]: I'd keep this as vanilla as possible... just personally

Comment (dgt41) [inline administrator/components/com_languages/views/override/tmpl/edit.php]: But this is jQuery, the line above the .on()...

Comment (dgt41) [inline administrator/components/com_languages/views/override/tmpl/edit.php]: And also the overrider.js is jQuery dependant

Comment (andrepereiradasilva) [inline administrator/components/com_languages/views/override/tmpl/edit.php]: http://vanilla-js.com/ :wink:  

IMHO Joomla core should move as most as possible to vanilla js for performance reasons.

But i also think that should be done when IE8, IE9 and IE10 are not supported by Joomla (4.0?).

Until them, if it's to much work to convert to vanilla js ... let's use jQuery.

Comment (wilsonge) [inline administrator/components/com_languages/views/override/tmpl/edit.php]: Yah Joomla 4 for that. Ie8 support for all of 3.x sadly

Comment (dgt41) [inline administrator/components/com_languages/views/override/tmpl/edit.php]: @andrepereiradasilva I suggest to look at: https://github.com/LeaVerou/bliss 
jQuery power at 8kb!!!

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 86a958a7c0d760d171b6a13c8fb4aed7dd416e17<br /><br />mootools is almost totally removed from Joomla.
great work @dgt41 !<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8926">issues.joomla.org/joomla-cms/8926</a>.</sub>

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 86a958a7c0d760d171b6a13c8fb4aed7dd416e17<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8926">issues.joomla.org/joomla-cms/8926</a>.</sub>

**Comment by infograf768** [comment]
RTC. Thanks.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8926">issues.joomla.org/joomla-cms/8926</a>.</sub>

**Comment by infograf768** [comment]
...<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8926">issues.joomla.org/joomla-cms/8926</a>.</sub>

**Comment by ssnobben** [comment]
"andrepereiradasilva added a note 10 days ago
http://vanilla-js.com/ :wink:
IMHO Joomla core should move as most as possible to vanilla js for performance reasons."

I had a fight for jquery v mootools long time ago with Joomla devs and that was a mistake done. So much trouble for incompatibility issues during the years for site developers.

Now though I think this well supported java script framework should be analyzed for quick implementation decision with just this argument --->performance reasons bcs situation is different. pls discuss/analyze this asap for the future plan bcs its so important for Joomla core functionality. Thanks.

**Comment by andrepereiradasilva** [comment]
I don't think Joomla should drop other frameworks support in the core. That should continue as it is.
Just say Joomla should use vanilla js where possible.

**Comment by ssnobben** [comment]
OK thats good as long as there is "no conflict" mess that have been a large deal breaker experience for many 3pds developers..

**Pull request merged**